roles/shiftstack/tasks/test_shiftstack.yml

```diff
@@ -82,19 +82,12 @@
 
   always:
     - name: Retrieve artifacts and installation files from the pod
```

Comment:
As the there is no loop any more I would remove the "installation files" from the task name.

Reply:
Done